sentry/src/test/java/io/sentry/SentryTracerTest.kt

```diff
@@ -143,70 +140,6 @@ class SentryTracerTest {
         verify(fixture.hub).setSpanContext(ex, tracer.root, "name")
     }
 
-    @Test
```

Comment:
TODO: maybe the tests from this class should be written by setting those fields to the Scope, so the transaction includes the fields